Vostok.Applications.AspNetCore/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using JetBrains.Annotations;
+using Microsoft.Extensions.DependencyInjection;
+using Vostok.Applications.AspNetCore.Configuration;
+using Vostok.Throttling;
+
+namespace Vostok.Applications.AspNetCore
+{
+    [PublicAPI]
+    public static class IServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds <see cref="DistributedContextSettings"/> to the specified service collection.
+        /// </summary>
+        public static IServiceCollection AddVostokDistributedContext(this IServiceCollection services)
+            => services.AddVostokDistributedContext(_ => {});
+
+        /// <summary>
+        /// Adds <see cref="DistributedContextSettings"/> to the specified service collection.
+        /// </summary>
+        public static IServiceCollection AddVostokDistributedContext(this IServiceCollection services, Action<DistributedContextSettings> setup)
+            => services.Configure(setup);
+
+        /// <summary>
+        /// Adds <see cref="DatacenterAwarenessSettings"/> to the specified service collection.
+        /// </summary>
+        public static IServiceCollection AddVostokDatacenterAwareness(this IServiceCollection services)
+            => services.AddVostokDatacenterAwareness(_ => {});
```

Comment:
Such overloads aren't necessary. Calls like `services.Configure<T>(_ => {})` actually do nothing as container always can create default instance of `IOptions<T>`. 

If default configuration is OK for user, middleware can be registered by simply writing `UseX()` in configure section.